src/pytom_tm/tmjob.py

```diff
@@ -551,6 +559,14 @@ def split_volume_search(self, split: tuple[int, int, int]) -> list[TMJob, ...]:
             )
 
         search_size = self.search_size
+        if self.tomogram_mask is not None:
+            tomogram_mask = read_mrc(self.tomogram_mask)
+            if np.all(tomogram_mask <= 0):
```

Comment:
This should skip a full tomogram if there is no positive region in the mask. I think now its only checked split_volume_search is ran on the TMJob, and that is not always done in parallel (https://github.com/SBC-Utrecht/pytom-match-pick/blob/fcdb82c28c29fea89a2a0582edf73a03e59e3f1d/src/pytom_tm/parallel.py#L86).
Maybe better to move it to the job initialization?

Reply:
I agree, decided to move it to the init of the TMJob instead